<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-03-17T14:36:37 IST" name="Default suite" finished-at="2024-03-17T14:37:08 IST" duration-ms="31114">
    <groups>
    </groups>
    <test started-at="2024-03-17T14:36:37 IST" name="Default test" finished-at="2024-03-17T14:37:08 IST" duration-ms="31114">
      <class name="practice.LoginFbWithSikuli">
        <test-method is-config="true" signature="lanuchFb()[pri:0, instance:practice.LoginFbWithSikuli@9d5509a]" started-at="2024-03-17T14:36:37 IST" name="lanuchFb" finished-at="2024-03-17T14:36:56 IST" duration-ms="18641" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- lanuchFb -->
        <test-method signature="loginzToFb()[pri:0, instance:practice.LoginFbWithSikuli@9d5509a]" started-at="2024-03-17T14:36:56 IST" name="loginzToFb" finished-at="2024-03-17T14:37:08 IST" duration-ms="12324" status="FAIL">
          <exception class="org.sikuli.script.FindFailed">
            <message>
              <![CDATA[D:/sikuli/userName.png: (944x364) in S(0)[0,0 1192x670] E:Y, T:3.0]]>
            </message>
            <full-stacktrace>
              <![CDATA[FindFailed: D:/sikuli/userName.png: (944x364) in S(0)[0,0 1192x670] E:Y, T:3.0
  Line 2761, in file Region.java

at org.sikuli.script.Region.wait(Region.java:2761)
at practice.LoginFbWithSikuli.loginzToFb(LoginFbWithSikuli.java:33)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.sikuli.script.FindFailed -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginzToFb -->
      </class> <!-- practice.LoginFbWithSikuli -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
